src/qt/rpcconsole.cpp

```diff
@@ -866,7 +866,7 @@ void RPCConsole::clear(bool keep_prompt)
     }
 
     // Set default style sheet
-    QFontInfo fixedFontInfo(GUIUtil::fixedPitchFont());
+    QFontInfo fixedFontInfo(GUIUtil::fixedPitchFont(true));
```

Comment:
To make code more readable and easier to reason about:
```suggestion
    QFontInfo fixedFontInfo(GUIUtil::fixedPitchFont(/*use_embedded_font=*/true));
```

See https://github.com/bitcoin/bitcoin/pull/23545

Reply:
@katesalazar @hebasto - thanks for the feedback!